gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -425,21 +425,28 @@ message VehiclePosition {
   }
   optional CongestionLevel congestion_level = 6;
 
-  // The degree of passenger occupancy of the vehicle or carriage. This field is still
-  // experimental, and subject to change. It may be formally adopted in the future.
+  // The nominal status of passenger occupancy for the vehicle or carriage.
+  // Individual producers may not publish all OccupancyStatus values. Therefore, consumers
+  // must not assume that the OccupancyStatus values follow a linear scale.
+  // Consumers should represent OccupancyStatus values as the nominal status indicated 
+  // and intended by the producer. Likewise, producers must use OccupancyStatus values that
```

Comment:
I assume localization is acceptable here, hence "represent" as the chosen wording?

Reply:
The use of the word "represent" is meant to convey what @barbeau described:
> a consumer shouldn't map a STANDING_ROOM_ONLY value to a roughly half-way full UI graphic bar. Instead, they should show the text "Standing room only" to end users.

Localization, such as translations, is acceptable as long as it does not change the meaning of the enum value.